tests/testnet.spec.js

```diff
@@ -747,11 +752,67 @@ describe('Transfer tokens', () => {
   it(`Check balance and balance change`, async () => {
     await timeout(10000)
     const result = await fioSdk2.genericAction('getFioBalance', {})
+    expect(result).to.have.all.keys('balance', 'available')
     fioBalanceAfter = result.balance
     expect(fundsAmount).to.equal(fioBalanceAfter - fioBalance)
   })
 })
 
+describe('Transfer locked tokens', () => {
+  const fundsAmount = FIOSDK.SUFUnit
+  let fioBalance = 0
+  let fioBalanceAfter = 0
+
+  it(`Check balance before transfer`, async () => {
+    const result = await fioSdk2.genericAction('getFioBalance', {})
+
+    fioBalance = result.balance
+  })
+
+  it(`Transfer locked tokens`, async () => {
+    try {
+      var mnem = FIOSDK.getMnemonic()
+      var privKey = await FIOSDK.createPrivateKeyMnemonic(mnem)
+      let pubkey =  FIOSDK.derivedPublicKey(privKey.fioKey)
+      const result = await fioSdk.genericAction('transferLockedTokens', {
+        payeePublicKey: pubkey.publicKey,
+        canVote: 0,
+        periods: [
+          {
+            duration: 120,
+            percent: 50,
+          },
+          {
+            duration: 240,
+            percent: 50,
+          }
+        ],
+        amount: fundsAmount,
+        maxFee: defaultFee,
+        tpid: '',
+      })
+
+
+      expect(result).to.have.all.keys('status', 'fee_collected')
+      expect(result.status).to.be.a('string')
+      expect(result.fee_collected).to.be.a('number')
+    }catch (e) {
+      console.log(e);
+    }
+  })
+
+  it(`Check balance and balance change`, async () => {
+    try {
+    await timeout(10000)
+    const result = await fioSdk2.genericAction('getFioBalance', {})
+    fioBalanceAfter = result.balance
+    expect(fundsAmount).to.equal(1000000000)
+    }catch (e) {
+      console.log(e);
+    }
+  })
+})
+
```

Comment:
would you add the same changes to the `index.spec.js`?

Reply:
yes, doing it.